cmd/yamlfmt/main.go

```diff
@@ -34,6 +38,15 @@ func run() error {
 	configureHelp()
 	flag.Parse()
 
+	if *flagVersion {
+		_, err := fmt.Printf("%s\n", version)
```

Comment:
By convention in this repo I'm not too worried about errors from `fmt.Printf`, so just doing the print here is fine.

I think this can also be simplified to `fmt.Println(version)` if I'm not mistaken.